canvas_modules/common-canvas/__tests__/common-properties/components/truncated-content-tooltip-test.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017-2022 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import TruncatedContentTooltip from "../../../src/common-properties/components/truncated-content-tooltip";
+import { mount } from "enzyme";
+import { expect } from "chai";
+
+
+describe("validation-message renders correctly", () => {
```

Comment:
This should be `Truncated content tooltip renders correctly`

Reply:
Copy error.  Thanks for finding.  Fixed.